### PR TITLE
Added message when google validations id different from enter validation

### PR DESCRIPTION
**Pull request by Kwasi008**
- check old address entered to what google validates

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```
Comment (micronix) [inline Gemfile:1]: hey @Kwasi008 why did you changed this to http?

**Review by micronix — CHANGES_REQUESTED** [review]
Hey @Kwasi008 what is this for? If I understand it correctly. When a user updates a ride and they change the address there is an alert that shows the address changed?

Also can you removed the commented out byebug statements

**Comment by Kwasi008** [comment]
Hi @micronix, I was working on Jira card 26 which was  "As an admin, I want to confirm riders' addresses after validation when creating rides". I can adjust it if that was the desired goal.

File: spec/requests/rides_spec.rb [hunk 2]
```diff
@@ -128,7 +128,6 @@
   it 'will change status to picking up on a ride for a driver' do
     post "/api/v1/rides/#{ride9.id}/picking-up",  headers: {"ACCEPT" => "application/json",  "Token" => "7897"}
     parsed_json = JSON.parse(response.body)
-    byebug
```
Comment (kidatsy) [inline spec/requests/rides_spec.rb:131]: So many old byebugs haha!

**Comment by Kwasi008** [comment]
Screen Shot 2021-01-22 at 2 39 00 PM

**Comment by Kwasi008** [comment]
Hi @micronix 
- changed how the flash message was working, we were not accounting for user alerts, we were just accounting for notice and errors and not alerts.
- Changed the controller's flash.alert into flash.errors when it is an error and not an alert.
- Attach a picture to show the changes

**Comment by Kwasi008** [comment]
@micronix 
- Added clear flash messages when the page is reloading

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -7,7 +7,20 @@ def pass_fail(application_state)
   def driver_active_inactive(is_active)
     is_active ? "Yes" : "No"
   end
-  
+
+  def bootstrap_class_for(flash_type)
+  case flash_type
+  when "notice"
+      "alert-success"
```
Comment (kidatsy) [inline app/helpers/application_helper.rb:14]: indentation?

Comment (Kwasi008) [reply app/helpers/application_helper.rb:14]: Thanks TC, I just went through and checked all of them. Please let me know if there anything else I should change on this PR. Thank you

**Review by jrmcgarvey — APPROVED** [review]
Looks OK to me.

**Pull request merged**